examples/typegraphs/execute.py

```diff
@@ -53,7 +53,7 @@ def roadmap(g: Graph):
     admins = deno.policy(
         "admins",
         """
-  (_args, { context }) => !!context.username
+  (_args, { context }) => !!context.username ? 'ALLOW' : 'DENY'
```

Comment:
suggestion: we'd probably want `ALLOW` : `PASS` in the examples to encourage composable policies.